plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -141,13 +144,21 @@ private String getJenkinsWar() {
 
         File pluginFileLocation = getPluginTxt();
         if (Files.exists(pluginFileLocation.toPath())) {
-            try (Scanner scanner = new Scanner(pluginFileLocation, StandardCharsets.UTF_8.name())) {
+            try (InputStreamReader inputStreamReader = new InputStreamReader(new FileInputStream(pluginFileLocation),
```

Comment:
You might be able to do this easier with streams with `BufferedReader.lines` ceated from `java.nio.file.Files`